ui/tests/acceptance/secrets/backend/ssh/ssh-test.js

```diff
@@ -112,13 +104,17 @@ module('Acceptance | ssh secret backend', function (hooks) {
     assert.dom('[data-test-ssh-configure-form]').exists('renders the empty configuration form');
 
     // default has generate CA checked so we just submit the form
-    await click('[data-test-ssh-input="configure-submit"]');
-
-    await waitFor('[data-test-ssh-input="public-key"]');
```

Comment:
This old waitFor was a little flaky locally, so opted to replace waitUntil with a specific timeout.